app/models/dog.rb

```diff
@@ -6,7 +6,8 @@ class Dog < ActiveRecord::Base
   has_many :observations
   has_many :views, class_name: "Observation", foreign_key: "observer_id"
   has_many :viewed_by,  class_name:  "Observation", foreign_key: "observed_id"
-  has_attached_file :img, storage: :s3, bucket: ENV['BUCKET_NAME']
+  has_attached_file :img, storage: :s3, bucket: ENV['BUCKET_NAME'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.